### ISSUE TITLE
Overwrite or merge changed data?

### ISSUE THREAD

**Issue by hofmand** [simon987/yt-metadata#1]
If `import.py` encounters a video that's already in the database with different data (e.g. the author has changed the title or description, or the `[video.view_count]` has increased, or the video has been deleted and so the `[video.title]` and other fields are blank), is it possible to keep both sets of data in the same database?

**Comment by simon987** [comment]
I'm not sure that I understand, what exactly would be the point of having the same video (with a different view count etc.) twice in the same database?
Right now if the same video ID exists the video is skipped (I think)

**Comment by hofmand** [comment]
The point of my question is to figure out how to keep the database synchronized with YouTube.com.

**Comment by simon987** [comment]
Right. I can definitely see the advantage of having multiple versions and analyzing the differences. 
An approach would be to tie every field that could potentially change to a version number, so that when a field changes, it could be appended to the database with a different version number (associated with a timestamp).
Did you have another idea in mind?

**Comment by hofmand** [comment]
Well that starts to get complicated. I don't have another idea in mind. But maybe it's not important if nobody else is asking for something like this.
